perl/p5-barcode-zbar/Portfile

```
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.branches      5.28 5.30 5.32
+perl5.setup         Barcode-ZBar 0.04
+
+platforms           darwin
+maintainers         nomaintainer
+license             {LGPL-2.1+}
+
+description         Barcode::ZBar (Perl interface to the ZBar Barcode Reader)
+
+long_description    ${description}
+
+checksums           size   57582 \
+                    rmd160 c48acc9fba0458f4a9572a9974c3af6bd409c726 \
+                    sha256 d57e1ad471b6a29fa4134650e6eec9eb834d42cbe8bf8f0608c67d6dd0f8f431
+
+depends_lib         port:zbar
+
+if {${perl5.major} != ""} {
+    depends_lib-append  port:perl${perl5.major}
+}
```

The `perl5 1.0` portgroup already specifies `depends_lib-append port:perl${perl5.major}`. The problem is that using `depends_lib zbar` instead of `depends_lib-append zlib` wipes out already-specified dependencie. The `zbar` dependency should also only apply to the `p5.xx-barcode-zbar` subports.

Nitpick: `ne` is preferred over `!=` for non-numeric exact string comparison. (I've proposed this change in the output of cpan2port: macports/macports-contrib#12)


```suggestion
if {${perl5.major} ne ""} {
     depends_lib-append  port:zlib
}
```